lib/active_merchant/billing/three_d_secure_eci_mapper.rb

```diff
@@ -0,0 +1,28 @@
+module ActiveMerchant
+  module Billing
+    class ThreeDSecureEciMapper
```

Comment:
Why do you need a class if you're not using anything on the instance? I think you can just declare `module ThreeDSecureEciMapper`.

Reply:
Changed to using a module